### ISSUE TITLE
Can't access admin panel.

### ISSUE THREAD

**Issue by michielhemme** [Mailu/Mailu#1056]
Hi in installed Mailu on kubernetes (baremetal).

When I try to access the admin site (https://mail.url.com/admin/ui) I get the following error:

Not Found

The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.

When looking at the logs of the admin container it shows that it is receiving the GET request.
10.244.0.1 - - [26/Jun/2019:12:39:26 +0000] "GET /admin/ui/ HTTP/1.1" 404 233 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:67.0) Gecko/20100101 Firefox/67.0"
10.244.0.1 - - [26/Jun/2019:12:40:27 +0000] "GET /admin/ui/ HTTP/1.1" 404 233 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:67.0) Gecko/20100101 Firefox/67.0"
10.244.0.1 - - [26/Jun/2019:12:40:27 +0000] "GET /admin/ui/ HTTP/1.1" 404 233 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:67.0) Gecko/20100101 Firefox/67.0"
10.244.0.1 - - [26/Jun/2019:12:40:28 +0000] "GET /admin/ui/ HTTP/1.1" 404 233 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:67.0) Gecko/20100101 Firefox/67.0"
10.244.0.1 - - [26/Jun/2019:12:40:28 +0000] "GET /admin/ui/ HTTP/1.1" 404 233 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:67.0) Gecko/20100101 Firefox/67.0"

normal webmail.url.com is working. And I can sent and receive mails on that client.

**Comment by kaiyou** [comment]
Did you enable the web admin in the settings?

**Comment by michielhemme** [comment]
###################################
# Optional features
###################################

# Expose the admin interface (value: true, false)                                                                      
ADMIN: "true"

# Path to the admin interface if enabled
# Kubernetes addition: You need to change ALL the ingresses, when you want this URL to be different!!!
WEB_ADMIN: "/admin"

**Comment by kaiyou** [comment]
It looks like your ingress is pointing directly at the admin container instead of the nginx who performs reverse proxying.

Normally the nginx would strip the /admin part which is not expected by the admin container.

**Comment by michielhemme** [comment]
Hmm oke.. I used the normal admin-ingress.yaml file but can't get it working.. 
Now i created my own and after your tip I can reach it by visiting:
http://admin.mydomain.com/ui :) Thanks

**Comment by trayla** [comment]
@michielhemme Could you please attach your admin-ingress.yaml? I ran into the same problem and cannot fix it for now. Thx!

**Comment by trayla** [comment]
The following configuration in admin-ingress.yaml worked for me. The first Ingress redirects the root path / to /ui/user/settings and the second Ingress receives any request below /ui and forwards it to the admin container. I don't know, if this is also suitable in other environments, but maybe it helps somebody.
```
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: mailu-admin-base
  namespace: mailu-mailserver
  annotations:
    kubernetes.io/ingress.class: nginx
    kubernetes.io/tls-acme: "true"
    nginx.ingress.kubernetes.io/permanent-redirect: /ui/user/settings
  labels:
    app: mailu
    role: mail
    tier: backend
spec:
  tls:
  - hosts:
    - mail.example.com
    secretName: letsencrypt-certs-all
  rules:
  - host: mail.example.com
    http:
      paths:
      - path: /
        backend:
          serviceName: admin
          servicePort: 80
---
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: mailu-admin-ui
  namespace: mailu-mailserver
  annotations:
    kubernetes.io/ingress.class: nginx
    kubernetes.io/tls-acme: "true"
  labels:
    app: mailu
    role: mail
    tier: backend
spec:
  tls:
  - hosts:
    - mail.example.com
    secretName: letsencrypt-certs-all
  rules:
  - host: mail.example.com
    http:
      paths:
      - path: /ui
        backend:
          serviceName: admin
          servicePort: 80
```

**Comment by srslafazan** [comment]
Hi, why was this closed? I just ran into this issue myself and fixed it with the help of @trayla's comment. The kubernetes ingresses are configured incorrectly, it seems that this should be re-opened.

Was it closed because of a fix after v1.7? The thread suggests that it's not fixed.

**Comment by kaiyou** [comment]
It was closed by the author himself. It was not reopen because the issue was fixed in master and the provided snippet is inconsistent with our current policy: the ingress should only point to the front service.
